Doc/library/sqlite3.rst

```diff
@@ -2269,8 +2339,67 @@ Explanation
 Transaction control
 ^^^^^^^^^^^^^^^^^^^
```

Comment:
When actually rendered, it looks like a mistake to have three section headers stacked up in a row with no words between them—especially since, inexplicably, with the legacy Python docs theme, h3 and h4 _are the same size_ (one more reason to dump the old theme as soon as practical).

Therefore, we need to add some text here, even if just a short summary. Here's a shot at it:

```suggestion
Transaction control
^^^^^^^^^^^^^^^^^^^

:mod:`!sqlite3` offers multiple methods of controlling whether, when and how
database transactions opened and closed.
:ref:`sqlite3-transaction-control-autocommit` is recommended,
while :ref:`sqlite3-transaction-control-isolation-level`
is consistent with pre-Python 3.12 versions of the module.
```

Reply:
Yeah, I thought about this myself. I'll add something.